tests/integration/integration.test.ts

```diff
@@ -30,7 +26,7 @@ it('outputs accessibility violation information for the demo app', (done) => {
  */
 function normalize(input: string) {
   /** Test times reported by Mocha. For example, `(520ms)` or `(3s)` */
-  const specTimePattern = /\(\d+m?s\)/g;
+  const specTimePattern = /\s*\([\d.]+m?s\)/g;
```

Comment:
The elapsed time can now have `.` in it, and this had to be updated to match.

Reply:
should it be `\.` instead of `.`?

Reply:
I originally thought it should, but it turns out that inside square brackets a period is a literal.

Our linting actually prevents us from writing this as `\.`, since the escaping is unnecessary.

Reply:
woah, TIL